wfe/wfe.go

```diff
@@ -623,12 +628,22 @@ func (wfe *WebFrontEndImpl) UpdateAccount(
 		ID:  existingAcct.ID,
 	}
 
-	newAcct.Contact = updateAcctReq.Contact
-	// Verify that the contact information provided is supported & valid
-	prob = wfe.verifyContacts(newAcct.Account)
-	if prob != nil {
-		wfe.sendError(prob, response)
+	switch {
+	case updateAcctReq.Status == deactivatedStatus:
+		newAcct.Status = updateAcctReq.Status
+	case updateAcctReq.Status != newAcct.Status:
+		wfe.sendError(
+			acme.MalformedProblem(fmt.Sprintf(
+				"Invalid account status: %s", updateAcctReq.Status)), response)
```

Comment:
I think this `%s` should be `%q`, otherwise sending `{ "status": "" }` returns `"Invalid account status: "`